frontend image upload returns 500 error

I've observed this on GKE when starting up a cluster, destroying it, and restarting the cluster, all without restarting the kiosk.

`kubectl logs frontend-deployment-[blahblah]-[blah]` shows this:
```
2018-11-12T17:02:48.276Z [ERROR] deepcell-20181111-1@deepcell-209717.iam.gserviceaccount.com does not have storage.objects.delete access to deepcell-output/uploads/img_000000000_FITC_001.png.
```

Just saw this myself, deploying on GKE:
```
2018-11-12T19:51:13.619Z [ERROR] training-test@deepcell-209717.iam.gserviceaccount.com does not have storage.objects.delete access to deepcell-output/uploads/mibi_nuclear.png.
Mon, 12 Nov 2018 19:51:13 GMT express:router <anonymous>  : /api/upload
2018-11-12T19:51:13.620Z [ERROR] middlewareError
Mon, 12 Nov 2018 19:51:13 GMT express:router <anonymous>  : /api/upload
2018-11-12T19:51:13.622Z [INFO] ::ffff:10.24.2.7 - - "POST /api/upload HTTP/1.1" 500 148 "http://104.196.231.104/predict" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36"

RangeError: Invalid status code: undefined
    at ServerResponse.writeHead (_http_server.js:199:11)
    at ServerResponse.writeHead (/usr/src/app/node_modules/on-headers/index.js:55:19)
    at ServerResponse.writeHead (/usr/src/app/node_modules/on-headers/index.js:55:19)
    at ServerResponse._implicitHeader (_http_server.js:190:8)
    at ServerResponse.end (/usr/src/app/node_modules/compression/index.js:103:14)
    at ServerResponse.send (/usr/src/app/node_modules/express/lib/response.js:221:10)
    at ServerResponse.json (/usr/src/app/node_modules/express/lib/response.js:267:15)
    at app.use (/usr/src/app/dist/server/config/express.js:95:26)
    at Layer.handle_error (/usr/src/app/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/usr/src/app/node_modules/express/lib/router/index.js:315:13)
```

--- a/conf/tasks/Makefile.gke
+++ b/conf/tasks/Makefile.gke
@@ -143,6 +143,7 @@ gke/create/service-account:
 ## Delete Service Account used by deepcell
 gke/destroy/service-account:
 	@echo "Destroying GKE service-account..."
+	@gcloud projects remove-iam-policy-binding $(CLOUDSDK_CORE_PROJECT) --member serviceAccount:$(GKE_NODE_SERVICE_ACCOUNT_EMAIL) --role roles/storage.admin
 	@-gcloud iam service-accounts delete $(GKE_NODE_SERVICE_ACCOUNT_EMAIL) --quiet
 	@echo "GKE service-account destruction finished."
 	@echo " "

